docs/eventing/samples/helloworld/helloworld-go/README.md

```diff
@@ -246,8 +246,10 @@ folder) you're ready to build and deploy the sample app.
       kubectl get broker --namespace knative-samples
       ```
 
-      _Note_: you can also use injection based on labels with the
-      [Eventing Sugar Controller](../../../../install/any-kubernetes-cluster.md).
+      **Note:** you can also use injection based on labels with the
+      Eventing Sugar Controller.
+      For how to install the Eventing Sugar Controller, see
```

Comment:
Small nit, I'd make "sugar controller" lowercase, but we don't have standards yet for things like this afaik
cc @omerbensaadon

Reply:
There has been a suggestion to have a word and phrase list. The docs are a bit inconsistent on capitalisation and I think having a list would make these decisions easier.